### PR TITLE
fixed search result with wrong bullet display and moved search result head…

### LINKED ISSUE #5915: [Docs] Formatting of search results includes an extra bullet
#### Current State
Search results on https://docs.meshery.io/search/ display an extraneous bullet.

#### Screenshot
Screen Shot 2022-07-29 at 2 31 47 PM

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by franklinekoh**
fixed search result with wrong bullet display

**fixed search result wrong bullet display and moved search result heading anchor to the right to prevent overlapping with search result**

This PR fixes #5915

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0e18537267f9581f8caa473cc46d324bf87acdc1 at: https://634556c1f163563afe012a31--meshery-docs-preview.netlify.app

File: docs/assets/css/anchorjs.css [hunk 1]
```diff
@@ -6,21 +6,24 @@
     color: inherit !important;
     text-decoration: none !important; /* do not underline */
     }
-    
+
+#search-results .anchorjs-link {
+    right: 0;
```
Comment (legendarykamal) [inline docs/assets/css/anchorjs.css:11]: we should find a way out , this isn't desired as it is getting unnoticed mostly in this case.
The anchor should be placed just right to the heading. 🤔

File: docs/_sass/overview.scss [hunk 2]
```diff
@@ -34,15 +34,20 @@ h2[id]:before, h4[id]:before, h5[id]:before {
     height: 6rem;
     visibility: hidden;
 }
+
+#search-results h4[id]:before {
```
Comment (legendarykamal) [inline docs/_sass/overview.scss:38]: is this change for the safari specific dot? wherever you are doing that change please mention in comments ,
that this is for safari extraneous dot appearance.

**Review by legendarykamal — CHANGES_REQUESTED** [review]
Please fix the dco too along with the following suggestions 👍 .
for dco check the contributing docs .

File: docs/_sass/overview.scss (hunk 2 above)
Comment (franklinekoh) [reply docs/_sass/overview.scss:38]: @legendarykamal just to confirm. You want me to mention that I am making the change for safari in this pull request comment? or do you mean in the code comment?

**Comment by leecalcote** [comment]
> Please fix the dco too along with the following suggestions 👍 .
> for dco check the contributing docs .

@legendarykamal, when contributors run into DCO challenges, it'd be good to point to both the immediate commands that can be run to address commit sign-off (in this PR - https://github.com/meshery/meshery/pull/6370/checks?check_run_id=8821564561), and also, point to the instructions that can get them setup long-term - https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin

File: docs/assets/css/anchorjs.css (hunk 1 above)
Comment (franklinekoh) [reply docs/assets/css/anchorjs.css:11]: I agree. The other way I can think of, that would get us the desired result will be to change the position of the DOM elements in html source code. I tried to look for the html code but couldn't find it. What would you suggest ?

File: docs/_sass/overview.scss (hunk 2 above)
Comment (leecalcote) [reply docs/_sass/overview.scss:38]: Commenting in the code is helpful in terms of sustainability.

**Comment by l5io** [comment]
🚀 Preview for commit 128b3104469af8f52f884ffc295d2656df02f8e8 at: https://634d2adb1304ae065ef0283f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 96aaff5d80556b001070a40306e8d0b9ab06167d at: https://634fc557ade0ee05517dc063--meshery-docs-preview.netlify.app

**Pull request closed without merging**